docs/_layouts/importer.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+---
+
+{% if page.render_limited_import_note %}
+<div class="note info">
+  <p>
+    This importer only converts your posts and creates YAML front-matter.
+    It does not import any layouts, styling, or external files (images, CSS, etc).
+  </p>
+</div>
+{% endif %}
+
+<h2>Invocation</h2>
+{% if page.prereq_info -%}
+<p>
+  {{ page.prereq_info | markdownify }}
+</p>
+{%- endif %}
+
+<p>
+  Sample snippet to invoke the importer:
+  <pre>jekyll import {{ page.cmd_name }}{% if page.cmd_opts %} {{ page.cmd_opts | map: 'switch' | join: ' ' }}{% endif %}</pre>
```

Comment:
[S9Y Database importer has a load of params (19, to be exact). This makes this slightly harder to see since there's so much horizontal scrolling.](https://deploy-preview-507--jekyll-import.netlify.app/docs/s9ydatabase/) Is it at all possible to make this a multiline output?

```text
jekyll import s9ydatabase \
  --dbname DB \
  --socket SOCKET \
...
```

Or do you think that would take up too much space? 🤔

Reply:
This block was just meant to quickly illustrate the CLI options. Technically for this importer, all of those options are *optional*. Therefore,
`jekyll import s9ydatabase` will work just as fine.
When making the block multiline, the end output is ugly (noisy) and we have to decide where exactly to draw the line to balance readability and accessibility. Besides, the table below lists all the options verbosely anyways.

Reply:
Ok makes sense. I thought they were all required when reading the example invocation. If there's a default value for the flag, maybe we can use the common convention of wrapping the argument in square brackets like `--socket [SOCKET]`?

Reply:
Modifying the long option here (i.e. bracketing the optional ARG) will cause a discrepancy between what's documented here and what gets generated in the terminal on invoking the CLI with `--help` switch.
Therefore, the best place to make this change is at the source-code level itself.